metrics-dropwizard/src/main/resources/reference.conf

```diff
@@ -0,0 +1,63 @@
+usi.metrics {
+
+  # A prefix that is used when constructing metric names
+  name-prefix: "usi"
+
+  # Histogram realated settings
+  histogram {
+    # The highest trackable value of histograms and timers
+    reservoir-highest-trackable-value: 3600000000000
+
+    # The number of significant decimal digits to which histograms and timers will maintain value resolution and separation
+    reservoir-significant-digits: 3
+
+    # Clear histograms and timers fully according to the given interval
+    reservoir-reset-periodically: true
+
+    # A histogram resetting interval
+    reservoir-resetting-interval: 5000ms
+
+    # Histogram reservoirs are divided into this number of chunks, and one chunk is cleared after each (resetting interval / number of chunks) elapsed
+    reservoir-resetting-chunks: 0
+  }
+
+  # Enable StatsD reporter. If enabled, the `statsd-reporter` block should be enabled and
+  # host and port should be provided
+  statsd-reporter: off
+
+  # StatsD reporter
+  # reporters.statsd {
+  #   # A remote hostname for the StatsD reporter
+  #   host:
```

Comment:
I like the solution of commenting this block since there is probably not a reasonable default.

Reply:
Yep, everything else doesn't seem to work with these type of settings